docs/topic_guides/evaluation/edge_metrics.md

```diff
@@ -0,0 +1,198 @@
+
+The Positive Predictive Value (Precision), is the proportion of predicted matches which are true matches.
+
+$$\textsf{Precision} = \frac{\textsf{True Positives}}{\textsf{All Predicted Positives}} = \frac{\textsf{True Positives}}{\textsf{True Positives} + \textsf{False Negatives}}$$
+
+??? info "Precision in Splink:"
+
+    - [accuracy_chart_from_labels_table](../../charts/accuracy_chart_from_labels_table.ipynb)
+    - [truth_space_table_from_labels_column](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_column)
+    - [truth_space_table_from_labels_table](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_table)
+    - [precision_recall_chart_from_labels_table](../../charts/precision_recall_chart_from_labels_table.ipynb)
+
+### Negative Predictive Value
+
+The Negative Predictive Value is the proportion of predicted non-matches which are true non-matches.
+
+$$\textsf{Negative Predictive Value} = \frac{\textsf{True Negatives}}{\textsf{All Predicted Negatives}} = \frac{\textsf{True Negatives}}{\textsf{True Negatives} + \textsf{False Positives}}$$
+
+??? info "Negative Predicitive Value in Splink:"
+
+    - [accuracy_chart_from_labels_table](../../charts/accuracy_chart_from_labels_table.ipynb)
+    - [truth_space_table_from_labels_column](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_column)
+    - [truth_space_table_from_labels_table](https://moj-analytical-services.github.io/splink/linkerqa.html#splink.linker.Linker.truth_space_table_from_labels_table)
+
+
+!!! warning 
+
+    Each of these metrics looks at just one row or column of the confusion matrix. A model cannot be meaningfully summarised by just one of these performance measures.
+
+    **“Predicts cancer with 100% Precision”** - is true of a “model” that correctly identifies one known cancer patient, but misdiagnoses everyone else as cancer-free.
+
+    **“AI judge’s verdicts have Recall of 100%”** - is true for a power-mad AI judge that declares everyone guilty, which would be stoically described by civil servants as “sub-optimal”.
+
+## Composite Metrics for Linkage
+
+This section contains composite metrics i.e. combinations of metrics that can been derived from the confusion matrix (Precision, Recall, Specificity and Negative Predictive Value). 
+
+Any comparison of two records has a number of possible outcomes (True Positives, False Positives etc.), each of which has a different impact on your specific use case. It is very rare that a single metric defines the desired behaviour of a model. Therefore, evaluating performance with a composite metric (or a combination of metrics) is advised.
+
+### F Score
+
+The F-Score is a weighted harmonic mean of Precision (Positive Predictive Value) and Recall (True Positive Rate). For a a general weight $\beta$:
```

Comment:
It might be nice to link through to wikipedia articles on these topics.

Reply:
Added